src/OpenLoco/Windows/VehicleList.cpp

```diff
@@ -27,36 +51,850 @@ namespace OpenLoco::Ui::Windows::VehicleList
+        makeWidget({ 1, 1 }, { 548, 13 }, widget_type::caption_24, 0),
+        makeWidget({ 535, 2 }, { 13, 13 }, widget_type::wt_9, 0, ImageIds::close_button, StringIds::tooltip_close_window),
+        makeWidget({ 0, 41 }, { 550, 172 }, widget_type::panel, 1),
+        makeRemapWidget({ 3, 15 }, { 31, 27 }, widget_type::wt_8, 1, ImageIds::tab, StringIds::tooltip_trains),
+        makeRemapWidget({ 3, 15 }, { 31, 27 }, widget_type::wt_8, 1, ImageIds::tab, StringIds::tooltip_buses),
+        makeRemapWidget({ 3, 15 }, { 31, 27 }, widget_type::wt_8, 1, ImageIds::tab, StringIds::tooltip_trucks),
+        makeRemapWidget({ 3, 15 }, { 31, 27 }, widget_type::wt_8, 1, ImageIds::tab, StringIds::tooltip_trams),
+        makeRemapWidget({ 3, 15 }, { 31, 27 }, widget_type::wt_8, 1, ImageIds::tab, StringIds::tooltip_aircraft),
+        makeRemapWidget({ 3, 15 }, { 31, 27 }, widget_type::wt_8, 1, ImageIds::tab, StringIds::tooltip_ships),
+        makeWidget({ 0, 14 }, { 26, 26 }, widget_type::wt_9, 0, StringIds::null, StringIds::tooltip_select_company),
+        makeWidget({ 4, 43 }, { 310, 12 }, widget_type::wt_14, 1, StringIds::null, StringIds::tooltip_sort_by_name),
+        makeWidget({ 314, 43 }, { 100, 12 }, widget_type::wt_14, 1, StringIds::null, StringIds::tooltip_sort_by_profit),
+        makeWidget({ 414, 43 }, { 65, 12 }, widget_type::wt_14, 1, StringIds::null, StringIds::tooltip_sort_by_age),
+        makeWidget({ 479, 43 }, { 67, 12 }, widget_type::wt_14, 1, StringIds::null, StringIds::tooltip_sort_by_reliability),
+        makeWidget({ 3, 56 }, { 544, 138 }, widget_type::scrollview, 1, scrollbars::vertical),
+        widgetEnd()
+    };
+
+    constexpr uint16_t _tabWidgets = (1 << Widx::tab_trains) | (1 << Widx::tab_buses) | (1 << Widx::tab_trucks) | (1 << Widx::tab_trams) | (1 << Widx::tab_aircraft) | (1 << Widx::tab_ships);
+    constexpr uint64_t _enabledWidgets = (1 << Widx::close_button) | _tabWidgets | (1 << Widx::company_select) | (1 << Widx::sort_name) | (1 << Widx::sort_profit) | (1 << Widx::sort_age) | (1 << Widx::sort_reliability) | (1 << Widx::scrollview);
+
+    enum SortMode : uint16_t
+    {
+        Name,
+        Profit,
+        Age,
+        Reliability,
+    };
+
+    static const uint8_t row_heights[] = {
+        28,
+        28,
+        28,
+        28,
+        48,
+        36
+    };
+
+    static Widx getTabFromType(uint8_t type);
+    static void initEvents();
+
+    // 0x004C1D4F
+    static void refreshVehicleList(window* self)
+    {
+        self->row_count = 0;
+        for (auto vehicle : EntityManager::VehicleList())
+        {
+            if (vehicle->vehicleType != static_cast<VehicleType>(self->current_tab))
+                continue;
+
+            if (vehicle->owner != self->number)
+                continue;
+
+            vehicle->var_0C &= ~Vehicles::Flags0C::sorted;
+        }
+    }
+
+    using VehicleHead = Vehicles::VehicleHead;
```

Comment:
`using Vehicles::VehicleHead;`
That way we bring it into scope rather than define a new symbol that is exactly the same but in this scope.